### ISSUE TITLE
Investigate issue with external module identities /enum found in IETF models

### ISSUE THREAD

**Issue by abhikeshav** [CiscoDevNet/ydk-gen#30]
When trying to use YDK on ```ietf-interfaces.yang```, there were issues encountered with identities defined in external modules (```iana-if-type.yang```). The prefix being used in the encoded RPC was wrong. Also, issues were seen with encoded enums in ```ietf-system.yang```.

**Comment by abhikeshav** [comment]
From Alberto:

> Possibly a bug: Use case: setting an identiyref. The identity is defined in a different module. The namespace of the module is not added. See below

> Thanks,
Alberto
        leaf type {
          description "Type of device group";
          tailf:info "Type of device group";
          type identityref {
            base "vts-ids:device-group-type";
          }
        }
What gets generated is:
```
  <server>
                    <name>VPC-GHOST</name>
                    <interface-name>eth0</interface-name>
                    <ip>1.2.3.4</ip>
                    <mac>dd:dd:dd:dd:dd:dd</mac>
                    <type>idx:virtual-server</type>
                  </server>
```
“Idx” is not defined though.
It should look something like:
```
                    <type xmlns:idx="http://cisco.com/ns/yang/vts/identities">idx:virtual-server</type>
```

**Comment by abhikeshav** [comment]
> FWIW, for the potential bug, I also see the symmetrical problem. I.e., when reading values that are identirefs (identities defined in a different module), the parsing fails:

>Thanks,
Alberto

```
2016-04-22 01:44:12,546 - ydk.services.CRUDService - INFO - READ operation completed
2016-04-22 01:44:12,552 - ydk.providers.NetconfServiceProvider - ERROR - Could not find identity tuple (cisco-vts, vts-ids:server) in identity_map, trying secondary mechanism
2016-04-22 01:44:12,552 - ydk.providers.NetconfServiceProvider - ERROR - Secondary mechanism failed no identity with name vts-ids:server found


Relevant snippet of the response:

<port>
  <name>Ethernet1/11</name>
  <connection-type xmlns:vts-ids="http://cisco.com/ns/yang/vts/identities">vts-ids:server</connection-type>
```

**Comment by abhikeshav** [comment]
Fixed in 0eb003c4e1024ed3bdd93d385a22507d8fb4e4a6
